### ISSUE TITLE
customized lable and icon not working in multiple-logbook-card

### ISSUE THREAD

**Issue by Spartacus68** [royto/logbook-card#117]
**Describe the bug**
it is not possible to configure the lable-value and the icon-value in parallel

**Expected behavior**
changen the lable of the device stat and the icon, depending on the state, also to overwrite the device name or to display the alias-name
**Screenshots**
![image](https://github.com/royto/logbook-card/assets/121975453/6c70312e-66bb-4c26-808f-555d041af75a)

**Environment (please complete the following information):**
 - HA Version: 2024.4.3
 - Browser: Edge
 - Card version: 2.5.5

**Card configuration**
Add your configuration here
```
type: custom:multiple-logbook-card
desc: true
entities:
  - entity: binary_sensor.reed_input
    state_map:
      - icon: mdi:door-open
        value: 'on'
      - icon: mdi:door-closed
        value: 'off'
      - label: geschlossen
        value: 'off'
      - label: geöffnet
        value: 'on'
  - entity: binary_sensor.ir_input
    state_map:
      - icon: mdi:leak
        value: 'on'
      - icon: mdi:leak-off
        value: 'off'
title: Haustür & Lichtschranke
```

**Comment by royto** [comment]
Hello @Spartacus68 

you need to define 1 state_map per value. If defined multiple time, only the first one is used.

use

```yaml
type: custom:multiple-logbook-card
desc: true
entities:
  - entity: binary_sensor.reed_input
    state_map:
      - icon: mdi:door-open
       label: geöffnet        
       value: 'on'
      - icon: mdi:door-closed
        label: geschlossen
        value: 'off'
  - entity: binary_sensor.ir_input
    state_map:
      - icon: mdi:leak
        value: 'on'
      - icon: mdi:leak-off
        value: 'off'
title: Haustür & Lichtschranke
```

From what I understand, you want to be able to customize the entity name, right ? When you talk about aliases, you mean assists aliases ? 
Can you create a separate issue for this.

**Comment by royto** [comment]
@Spartacus68 did you have the time to test using the solution provided ?

**Comment by Spartacus68** [comment]
@royto I checked this and yes, this is ok. But you can only change the icon **OR** the values, not both.

Means if I would like to change the icon and the value of an entity depending on its status, the only the first mapping (under state_map) will processed.

**Comment by royto** [comment]
You can change both using provided configuration ... 
Did you test it ?

**Comment by royto** [comment]
I update the configuration in the previous response (should be valid now)

To change both icon and label for the SAME entity, set it in the **same** state map:
```yaml
...
    state_map:
      - icon: mdi:door-open
         label: geöffnet        
         value: 'on'
 ...
```yaml

**Comment by Spartacus68** [comment]
ok! I have it, seems to be working now. 

Is there also a way to overwrite the entity name? The entity is a reed-contact  but it makes more sense to say that the door is open.
In my case, I would like to overwrite the entity name (Haustür Reedkontakt) with "Haustür"!
![image](https://github.com/royto/logbook-card/assets/121975453/e6fc4523-ed66-4e7c-8c41-87ba16d0085d)

sorry for the confusion at the beginning!
Spartacus

**Comment by royto** [comment]
I created a dedicated [issue](https://github.com/royto/logbook-card/issues/120) for this feature request. Closing this one as the problem is solved

**Comment by royto** [comment]
@Spartacus68 You can already specify a custom label for entity using the `label` option ...

```yaml
type: custom:multiple-logbook-card
desc: true
entities:
  - entity: binary_sensor.reed_input
    label: 'Haustür'
    state_map:
      - icon: mdi:door-open
        value: 'on'
      - icon: mdi:door-closed
        value: 'off'
      - label: geschlossen
        value: 'off'
      - label: geöffnet
        value: 'on'
  - entity: binary_sensor.ir_input
    state_map:
      - icon: mdi:leak
        value: 'on'
      - icon: mdi:leak-off
        value: 'off'
title: Haustür & Lichtschranke
```

**Comment by Spartacus68** [comment]
@royto 
Hi,
thank you very much! It works!
